lib/test/test-utils.ts

```diff
@@ -328,7 +332,7 @@ const runComponentTest = ({
                     // and use APCA bronze level instead
```

Comment:
This comment needs updating (e.g. we aim for bronze but at the moment we had to settle for less strict thresholds)